README.md

```diff
@@ -87,6 +87,43 @@ export default defineConfig({
 });
 ```
 
+<details><summary>notes for vite projects</summary>
+
+  If you're `import.meta.glob`ing large chunks of your codebase, you'll want to make sure that your globs exclude CSS files
+
+  For example:
+
+
+  ❌ Don't do this
+  ```js
+  ...import.meta.glob('./templates/**/*', { eager: true })
+  ```
+
+  ✅ Do this
+  ```js
+  ...import.meta.glob('./templates/**/*.{gjs,gts}', { eager: true }),
+  ```
+  or better yet, for small projects:
+  ```js
+  ...import.meta.glob('./templates/{top,level,folders}/{sub,folders}.{gjs,gts}', { eager: true }),
+  ```
```

Comment:
Looks like the correct fix, but in the wrong location: https://github.com/ember-cli/ember-strict-application-resolver#usage - this is where the copypasta is coming from.